### PR TITLE
Refactored Person files in backend to include interviewer job qualifications

### PR DESCRIPTION
**Pull request by apluscs**

### CODE REVIEW

File: src/test/java/com/google/sps/data/DatastorePersonDaoTest.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    dao = new DatastorePersonDao();
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  // Checks that an Person is stored in datastore.
```
Comment (meagancheese) [inline src/test/java/com/google/sps/data/DatastorePersonDaoTest.java]: NIT: a, not an

File: src/test/java/com/google/sps/data/DatastorePersonDaoTest.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            a.firstName(),
+            a.lastName(),
+            a.company(),
+            a.job(),
+            a.linkedIn(),
+            a.qualifiedJobs());
+    Assert.assertEquals(personAWithID, storedPerson);
+  }
+
+  // Checks that an Person is updated in datastore.
```
Comment (meagancheese) [inline src/test/java/com/google/sps/data/DatastorePersonDaoTest.java]: Same here (a, not an).

**Review by meagancheese — APPROVED** [review]
Super small nits, otherwise LGTM

**Comment by meagancheese** [comment]
Also, you should make sure the Person class is not called anywhere else / run the tests to see if anything breaks.

**Review by GrantComm — APPROVED** [review]
LGTM

**Comment by apluscs** [comment]
@meagancheese Yep, the green checks mean Git ran the tests already and they passed. Rn there's no issues with the backend itself, but there are some with communication between front/backend (like how the JSON object is being put together in JS)

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**